terraform/variables.tf

```diff
@@ -45,8 +45,3 @@ variable "memory" {
   description = "Memory to allocate in MB. Powers of 2 only."
   type        = number
 }
-
-variable "beta_search_enabled" {
```

Comment:
We can get rid of the var in the corresponding `config_<environment>.tfvars` files now, too

Reply:
Done